accelerometer: true
gyroscope: false

real_time_fft: false